src-json/define.json

```diff
@@ -0,0 +1,584 @@
+[
+  {
```

Comment:
Probably makes more sense to use tabs for indentation? That's what all the other sources use (ml, hx, or even other JSON files like in `.vscode`).